server/src/codeActions.ts

```diff
@@ -373,16 +375,26 @@ let handleUndefinedRecordFieldsAction = ({
           newText += paddingContentRecordField;
         }
 
-        newText += `${fieldName}: failwith("TODO"),\n`;
+        newText += `${fieldName}: ${todoValue},\n`;
       });
 
       // Let's put the end brace back where it was (we still have it to the direct right of us).
       newText += `${paddingContentEndBrace}`;
     } else {
       // A single line record definition body is a bit easier - we'll just add the new fields on the same line.
-      newText += ", ";
+
+      // For an empty record (`range.end.character - range.start.character == 2`),
+      // we don't want to add an initial trailing comma as that would be invalid syntax.
+      //
+      // We assume that records that already contain some characters between
+      // their braces have at least one field and therefore we need to insert
+      // an initial trailing comma.
+      if (range.end.character - range.start.character > 2) {
+        newText += ", ";
+      }
```

Comment:
Bug fix 2: initial comma was inserted when using this code action on empty records, resulting in invalid syntax